ADOPTERS.md

```diff
@@ -21,3 +21,7 @@ This is a list of adopters of using PowerShell in production or in their product
 * [PowerShell Universal Dashboard](https://ironmansoftware.com/powershell-universal-dashboard) is a cross-platform web framework for PowerShell. 
   It provides the ability to create robust, interactive websites, REST APIs, and Electron-based desktop apps with PowerShell script. 
   More information about PowerShell Universal Dashboard is available at the [PowerShell Universal Dashboard Docs](https://docs.universaldashboard.io).
+* [System Frontier](https://systemfrontier.com/solutions/powershell/) provides dynamically generated web GUIs and REST APIs for PowerShell and other scripting languages.  
```

Comment:
Static analysis failed on this. 
You need to add `REST` in https://github.com/PowerShell/PowerShell/blob/master/.spelling file.
Also you could add `GUIs` or reword.

Reply:
Added both REST and GUIs.